### DISCUSSION TITLE
Can't hack my camera pls help

### DISCUSSION THREAD

**Discussion by IngYordiDelgado** [roleoroleo/yi-hack-Allwinner-v2#923] [General]
Hi I have a camera with the following board I try to make a backup with the files provided but didn't work also try the hack b091bq but it didn't work too when I try to go to the ip that my router assign I can't access anything also when the SD is inside the yi app stop connecting to the camera. When I remove the SD the camera work ok again. 
![IMG20240823174917](https://github.com/user-attachments/assets/efa8d564-98f5-483c-9b6d-757eb237d382)
![IMG20240823175200](https://github.com/user-attachments/assets/6531c49a-16b1-421b-8220-0ae2222db181)

**Comment by gl2007** [comment]
What is the size of your sd card? Have you tried with a different branded small size sd card like 8GB or 16GB, assuming this one is a bigger one?

**Reply by IngYordiDelgado** [reply]
I try with 32gb card and I just try with a 4gb old one no look neither

**Comment by roleoroleo** [comment]
Could you please take a picture of the whole pcb?
What fw version do you read in the app?

**Comment by IngYordiDelgado** [comment]
![IMG-20240914-WA0019](https://github.com/user-attachments/assets/07486f45-7df0-48f2-bd48-74b87d4be517)
![Screenshot_2024-09-14-17-28-56-21_e1e189ae9b7c42b78424826bcc97b70a](https://github.com/user-attachments/assets/c0ccea56-5a7d-43f0-bebe-7bff75215200)

**Comment by roleoroleo** [comment]
Your cam is not supported at the moment.
Please, send me a backup of the flash and I will try to support it.
